scss/jquery/overrides/_select2.scss

```diff
@@ -0,0 +1,246 @@
+          }
+        }
+      }
+      /* stylelint-enable selector-max-compound-selectors, max-nesting-depth */
+    }
+
+    // The input field overlay
+    .select2-search-field {
+      /* stylelint-disable selector-max-compound-selectors */
+      input {
+        @include open-sans-font-family();
+        // These !important are necessary to counteract Bootstrap's base style
+        // when select2 is used inside the bootstrap namespace
+        font-size: $font-size-base !important;
+        line-height: normal !important;
+        margin: 0;
+        padding: 4px 4px 6px 8px;
+
+        // Spinner icon that appears when fetching data via ajax
+        &.select2-active { /* stylelint-disable-line max-nesting-depth, selector-no-qualifying-type */
+          background-position: center right 20px !important;
+        }
+      }
+      /* stylelint-enable selector-max-compound-selectors */
+    }
+  }
+}
+
+// The state of both single and multiple select2 when open
+.select2-dropdown-open {
+  .select2-choice,
+  .select2-choices, {
+    border-bottom-left-radius: 0;
+    border-bottom-right-radius: 0;
+    border-color: $input-border-focus;
+    box-shadow: none;
+  }
+
+  // Only for single select2, when open the arrow points up
+  .select2-choice {
+    .select2-arrow {
+      &::before {
+        content: $fa-var-caret-up;
+      }
+    }
+  }
+}
+
+// The select2 dropdown is appended right before </body>
+#{$outside-namespace-marker} {
+  // The dropdown list
+  .select2-drop.select2-drop-active.crm-container {
+    background: $crm-white;
+    border-bottom-left-radius: $border-radius-base;
+    border-bottom-right-radius: $border-radius-base;
+    border-color: $input-border-focus;
+
+    // CiviCRM-specific "refine search" controls
+    .crm-entityref-filters {
+      /* stylelint-disable selector-max-compound-selectors */
+      select,
+      input {
+        background: $crm-white;
+        border-color: $crm-grayblue-darker;
+        box-sizing: border-box;
+        height: $input-height-base;
+      }
+      /* stylelint-enable selector-max-compound-selectors */
+    }
+
+    // The list of items that match the search
+    .select2-results {
+      /* stylelint-disable selector-max-compound-selectors */
+      li {
+        color: $gray-dark;
+        padding: 5px;
+      }
+
+      // Class applied to an item when hovered
+      .select2-highlighted {
+        background: $gray-light;
+
+        p { /* stylelint-disable-line max-nesting-depth */
+          color: $gray-dark;
+        }
+      }
+
+      .select2-result-label {
+        padding: 5px 10px;
+      }
+      /* stylelint-enable selector-max-compound-selectors */
+    }
+  }
+
+  // Search text input field
+  .select2-search {
+    &::before {
+      @include fa-icon();
+      color: $gray-darker;
+      content: $fa-var-search;
+      font-size: $font-size-base;
+      position: absolute;
+      right: 10px;
```

Comment:
Same as above.

Reply:
it's simply to align the search icon in the search field

<img width="200" alt="icon" src="https://user-images.githubusercontent.com/6400898/46878457-a4ee2000-ce43-11e8-9026-ec8d78817334.png">